DigitalLearningSolutions.Data/DataServices/SupervisorService.cs

```diff
@@ -216,6 +218,34 @@ public SupervisorDelegateDetail GetSupervisorDelegateDetailsById(int supervisorD
                ).FirstOrDefault();
         }
 
+        public int? ValidateDelegate(int centreId, string delegateEmail)
+        {
+            int? delegateUserId = (int?)connection.ExecuteScalar(

```

Comment:
If we are using `ExecuteScalar` - can we be certain that the query will only return 1 result?